website/content/api-docs/secret/identity/mfa/login-enforcement.mdx

```diff
@@ -0,0 +1,152 @@
+---
+layout: api
+page_title: /identity/mfa/login-enforcement - HTTP API
+description: >-
+  The '/identity/mfa/login-enforcement' endpoint focuses on specifying which MFA methods should be used
+  when logging in.
+---
+
+## Create a login enforcement
+
+This endpoint creates or updates a login enforcement that specifies which MFA methods should be used when logging into
```

Comment:
Should there be a note on how multiple enforcements on the same scope interact i.e that they get ANDed together?

Reply:
Ah yes, good idea.